src/main/java/seedu/address/model/SeplendidModelManager.java

```diff
@@ -56,20 +56,27 @@ public class SeplendidModelManager implements SeplendidModel {
      * Initializes a SeplendidModelManager with the given localCourseCatalogue, userPrefs,
      * partnerCourseCatalogue, universityCatalogue, mappingCatalogue, noteCatalogue.
      */
-    public SeplendidModelManager(ReadOnlyLocalCourseCatalogue localCourseCatalogue, ReadOnlyUserPrefs userPrefs,
+    public SeplendidModelManager(ReadOnlyUserPrefs userPrefs,
+                                 ReadOnlyLocalCourseCatalogue localCourseCatalogue,
                                  ReadOnlyPartnerCourseCatalogue partnerCourseCatalogue,
                                  ReadOnlyUniversityCatalogue universityCatalogue,
-                                 ReadOnlyNoteCatalogue noteCatalogue,
-                                 ReadOnlyMappingCatalogue mappingCatalogue) {
+                                 ReadOnlyMappingCatalogue mappingCatalogue,
+                                 ReadOnlyNoteCatalogue noteCatalogue) {
         requireAllNonNull(localCourseCatalogue, userPrefs, partnerCourseCatalogue, universityCatalogue,
                 noteCatalogue, mappingCatalogue);
 
-        logger.fine("Initializing with local course catalogue: " + localCourseCatalogue
-                + ",\npartner course catalogue: " + partnerCourseCatalogue
-                + ",\nuniversity catalogue: " + universityCatalogue
-                + ",\nmapping catalogue: " + mappingCatalogue
-                + ",\nnote catalogue: " + noteCatalogue
-                + "\n and user prefs " + userPrefs);
+        logger.fine(String.format("Initializing with user prefs: %s,\n"
+                                  + "local course catalogue: %s, \n"
+                                  + "partner course catalogue: %s,\n"
+                                  + "university catalogue: %s,\n"
+                                  + "mapping catalogue: %s,\n"
+                                  + "and note catalogue: %s",
+                                  userPrefs,
+                                  localCourseCatalogue,
+                                  partnerCourseCatalogue,
+                                  universityCatalogue,
+                                  mappingCatalogue,
+                                  noteCatalogue));
```

Comment:
This might contradict the coding style guide:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.

Not too sure though, please help review this if you have time. Thanks!

Reply:
Oh, for strings concat, it's allowed to be on the same indentation right? I see why you changed it.